components/service/glean/README.md

```diff
@@ -124,6 +127,33 @@ the following command can be used:
 
 `adb shell am start -n org.mozilla.samples.glean/mozilla.components.service.glean.debug.GleanDebugActivity --ez logPings true`
 
+### Debug intent receiver
+
+Glean also contains an intent receiver that can be sent intents (messages) to
+perform certain actions in a running application.
+
+Users can trigger these intents using the following
+[`adb`](https://developer.android.com/studio/command-line/adb) command:
+
+`adb shell am broadcast -a {NAME_OF_INTENT} [extra keys]`
+
+The following intents are supported:
+
+- `org.mozilla.components.service.glean.sendPing`: Sends a ping immediately.
+
+  It has two parameters:
+  - pingName: The name of the ping
+  - docType: The docType to use in the URL (optional: if not provided,
```

Comment:
Do we have a specific use-case for this? I can't think of any off the top of my head: I'd vote for dropping this :)

Reply:
We will when we have custom pings where the user will probably put things on a custom ping but send to a generic doctype endpoint.

Reply:
Do we even allow that? A specific custom ping name should end up in a specific doctype endpoint, not a generic one. Moreover, even if we were to do that, tagging in the debug view will address this?

Reply:
+1 for this!

Reply:
I was going off of the signature for `sendPing(pingName, docType)`.  Maybe we should refactor that as well?  We currently only ever use it with `pingName == docType`.

Reply:
Yeah, we should probably refactor that: I don't think a future dev-facing API would allow for custom doctype != pingName.

Reply:
Ok.  I've filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1531038) for that.  Should be simple enough, but we don't need to confuse this PR with that.